api/src/utils/validate-storage.ts

```diff
@@ -0,0 +1,31 @@
+import env from '../env';
+import logger from '../logger';
+import { access } from 'fs/promises';
```

Comment:
Importing from `fs/promises` is only possible since Node v14 according to [this](https://nodejs.org/api/fs.html#fs_promises_api). We should probably just use `fs-extra` here as we are doing in _most_ other places working with the filesystem.

Reply:
ref 5b92d7fa6